frontend/src/App.tsx

```diff
@@ -31,9 +32,8 @@ const App = (): React.ReactElement => {
     AUTHENTICATED_USER_KEY,
   );
 
-  const [authenticatedUser, setAuthenticatedUser] = useState<AuthenticatedUser>(
-    currentUser,
-  );
+  const [authenticatedUser, setAuthenticatedUser] =
```

Comment:
Can you take a look at this warning? I'm not sure what it's trying to point out so if it's not important just ignore it. Also did you run the linters before pushing?

Reply:
I ran the front-end linter, but not the back-end linter. I'll run both and push